Libplanet/Net/BoundPeer.cs

```diff
@@ -53,6 +54,58 @@ private BoundPeer(SerializationInfo info, StreamingContext context)
         public static bool operator !=(BoundPeer left, BoundPeer right) =>
             Operator.Weave(left, right);
 
+        /// <summary>
+        /// Initializes a new instance of the <see cref="BoundPeer"/> class from
+        /// comma-separated string.</summary>
+        /// <param name="peerInfo">A comma-separated string have format {pubkey},{host},{port}.
+        /// </param>
+        /// <returns>A <see cref="BoundPeer"/> from given data.</returns>
```

Comment:
Could you add [`exception` XML description](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/xmldoc/exception) for `ArgumentException`?

Reply:
Amended. https://github.com/planetarium/libplanet/pull/1240/commits/76e98125136f76e43a925a7e865f0cd04b8aaa7d